__tests__/_/binaries.js

```diff
@@ -8,7 +8,7 @@ const zlib = require('zlib');
 
 
 export const p = os.platform();
-export const bv = (binaries_version || version).split('.')[0];
+export const bv = process.env.TON_SDK_BIN_VERSION ? process.env.TON_SDK_BIN_VERSION : (binaries_version || version).split('.')[0];
```

Comment:
```suggestion
export const bv = process.env.TON_SDK_BIN_VERSION || binaries_version || version.split('.')[0];
```

Reply:
Should it be 
`process.env.TON_SDK_BIN_VERSION || (binaries_version || version).split('.')[0]` ?
Cause `binaries_version` also should be truncated to major number, as far as i understand

Reply:
Fixed in last commit, +removed `binaries_version` from package as obsolete